x-pack/legacy/plugins/canvas/public/applications/canvas.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Store } from 'redux';
+import ReactDOM from 'react-dom';
+import { I18nProvider } from '@kbn/i18n/react';
+import { Provider } from 'react-redux';
+
+import { AppMountParameters, CoreStart } from 'kibana/public';
+
+// @ts-ignore Untyped local
+import { App } from '../components/app';
+import { KibanaContextProvider } from '../../../../../../src/plugins/kibana_react/public';
+
+export const renderApp = (
+  coreStart: CoreStart,
+  plugins: object,
+  { element }: AppMountParameters,
+  canvasStore: Store
+) => {
+  ReactDOM.render(
+    <KibanaContextProvider services={{ ...coreStart, ...plugins }}>
+      <I18nProvider>
+        <Provider store={canvasStore}>
+          <App />
+        </Provider>
+      </I18nProvider>
+    </KibanaContextProvider>,
+    element
+  );
+  return () => ReactDOM.unmountComponentAtNode(element);
+};
```

Comment:
This was created based on the new conventions laid out here: https://github.com/elastic/kibana/blob/0dac43516e0690486d13071063cb089bce96ee11/src/core/CONVENTIONS.md#applications

The thing is, it's kinda weird to have `applications` at the same level as our `apps` directory (which houses home, workpad, and export)